x/multisig/tx.go

```diff
@@ -0,0 +1,16 @@
+package multisig
+
+import (
+	"github.com/iov-one/weave"
+)
+
+// MultiSigTx is an optional interface for a Tx that allows
+// it to support multisig contract
+type MultiSigTx interface {
+	GetMultisig() []byte
+}
+
+// MultiSigCondition returns condition for a contract ID
```

Comment:
I would move these lines into context.go, where I was looking for them

Then MultiSigTx could also move into msg.go? Or stay here